how-to/security/install-openvpn.md

```diff
@@ -438,6 +438,46 @@ sudo systemctl restart openvpn@client
 
 You should now be able to connect to the fully remote LAN through the VPN.
 
+### Using OpenSSL providers
+
+OpenVPN uses the OpenSSL 3 Default Provider on Ubuntu. However, you can include additional providers dynamically depending on your use case.
+
+#### Legacy provider
+
+In Ubuntu 24.04 LTS and later, you can still use legacy algorithms by explicitly adding them through the `legacy` provider alongside `default`. To do this, add the following line to your configuration.
+
+```
+providers legacy default
+```
+
+You can also run openvpn with the `--providers` argument.
+
+```bash
+openvpn --providers legacy default ...
```

Comment:
the list of providers probably need to be quoted? Like `--providers "legacy default ..."`